### PR TITLE
fix creating sessions

### PR DESCRIPTION
**Pull request by KazuCocoa**
Fix createSession to work after https://github.com/appium/appium/pull/9791

### CODE REVIEW

File: lib/appium_lib_core/common/base/bridge.rb [hunk 1]
```diff
@@ -24,6 +24,19 @@ def self.handshake(**opts)
             raise CoreError, 'cannot understand dialect'
           end
         end
+
+        private
+
+        # Use capabilities directory because Appium's capability is based on W3C one.
+        # Called in bridge.create_session(desired_capabilities)
+        def merged_capabilities(desired_capabilities)
```
Comment (KazuCocoa) [inline lib/appium_lib_core/common/base/bridge.rb]: selenium-webdriver

```ruby
        def merged_capabilities(oss_capabilities)
          w3c_capabilities = W3C::Capabilities.from_oss(oss_capabilities)

          {
            desiredCapabilities: oss_capabilities,
            capabilities: {
              firstMatch: [w3c_capabilities]
            }
          }
        end
```

**Comment by KazuCocoa** [comment]
The reason why Ruby client filter the capabilities are valid only for the following capabilities:

- https://www.w3.org/TR/webdriver/#dfn-validate-capabilities
- https://github.com/SeleniumHQ/selenium/blob/0433610b6fc8249397c0eb5eb0e62614405afdd6/rb/lib/selenium/webdriver/remote/w3c/capabilities.rb#L34

```
          KNOWN = [
            :browser_name,
            :browser_version,
            :platform_name,
            :accept_insecure_certs,
            :page_load_strategy,
            :proxy,
            :set_window_rect,
            :timeouts,
            :unhandled_prompt_behavior,

            # remote-specific
            :remote_session_id,

            # TODO (alex): deprecate in favor of Firefox::Options?
            :accessibility_checks,
            :device,

            # TODO (alex): deprecate compatibility with OSS-capabilities
            :implicit_timeout,
            :page_load_timeout,
            :script_timeout,
          ].freeze
```

**Comment by KazuCocoa** [comment]
📝 

add `appium:` prefix for to pass:

https://github.com/SeleniumHQ/selenium/blob/0433610b6fc8249397c0eb5eb0e62614405afdd6/rb/lib/selenium/webdriver/remote/w3c/capabilities.rb#L158

instead of skipping `from_oss`

**Comment by dpgraham** [comment]
I'm going to make a PR for Appium today as well that merges `desiredCapabilities` into the `alwaysMatch` W3C object.

**Pull request merged**